code/game/objects/structures/janicart.dm

```diff
@@ -6,72 +6,138 @@
 	anchored = 0
 	density = 1
 	flags = OPENCONTAINER
+	climbable = TRUE
 	//copypaste sorry
 	var/amount_per_transfer_from_this = 5 //shit I dunno, adding this so syringes stop runtime erroring. --NeoFite
 	var/obj/item/weapon/storage/bag/trash/mybag	= null
 	var/obj/item/weapon/mop/mymop = null
 	var/obj/item/weapon/reagent_containers/spray/myspray = null
 	var/obj/item/device/lightreplacer/myreplacer = null
+	var/obj/structure/mopbucket/mybucket = null
+	var/has_items = FALSE
+	var/dismantled = TRUE
 	var/signs = 0	//maximum capacity hardcoded below
 
 
 /obj/structure/janitorialcart/New()
-	create_reagents(460)
+	//create_reagents(460)
```

Comment:
BTW `..()` is missed here. Or may be it's better to remove/comment that New ovveride completely